testsuite/features/secondary/allcli_system_group.feature

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2017-2022 SUSE LLC
 # Licensed under the terms of the MIT license.
 
+@skip_if_github_validation
```

Comment:
In this case, maybe you can just add the tag in `Scenario: Install the locale formula package on the server` as you do with `@skip_if_container_server` ?

Reply:
Don't the tests after that scenario depend on having the package installed? It is skipped for the container server because the container server contains that RPM already... Actually, the package is installed in the container, but it is the one from the last uyuni release.

Reply:
But for your purpose.. it should be the same right?
`@skip_if_github_validation` and `@skip_if_container_server` both work with server container image, doesn't them?